data/project-templates/js-node/src/index.js

```diff
@@ -0,0 +1,39 @@
+import * as dotenv from "dotenv";
+import { LogLevel, ProposalFilters, TaskExecutor } from "@golem-sdk/golem-js";
+
+dotenv.config();
+
+(async function main() {
+  const executor = await TaskExecutor.create({
+    // What do you want to run
+    package: "golem/node:20-alpine",
+
+    // How much you wish to spend
+    budget: 0.5,
+    proposalFilter: ProposalFilters.limitPriceFilter({
+      start: 0.1,
+      cpuPerSec: 0.1 / 3600,
+      envPerSec: 0.1 / 3600,
+    }),
+
+    // Where you want to spend
+    payment: {
+      network: "polygon",
+    },
+
+    // Control the execution of tasks
+    maxTaskRetries: 0,
+
+    // Useful for debugging
+    logLevel: LogLevel.Info,
+    taskTimeout: 5 * 60 * 1000,
+  });
+
+  try {
+    // Your code goes here
```

Comment:
maybe we should give an example of using `executor.run` and `ctx` ??

Reply:
This is copied from the example supplied by Grzesiek - don't know if we should be making changes here